### PR TITLE
ONEMPERS-358 added api definition

### PR DESCRIPTION
**Pull request by smiasojed**

### CODE REVIEW

File: LgiHdmiCec/LgiHdmiCec.json [hunk 1]
```diff
@@ -57,26 +120,23 @@
...
-                                            "type": "integer",
-                                            "example": 3
-                                        }
-                                    },
-                                    "required": [
-                                        "deviceType",
-                                        "logicalAddress"
-                                    ]
-                                }
+				  "type":"object",
```
Comment (michalkucharczyk) [inline LgiHdmiCec/LgiHdmiCec.json:123]: indentation

File: LgiHdmiCec/LgiHdmiCec.json [hunk 2]
```diff
@@ -145,7 +205,116 @@
...
+                },
+                "required": [
+                    "actionName"
+                ]
+            },
+            "result": {
+                "$ref": "#/definitions/result"
+            }
+        },
+	"setPingInterval":{
```
Comment (michalkucharczyk) [inline LgiHdmiCec/LgiHdmiCec.json:243]: indentation

File: LgiHdmiCec/LgiHdmiCec.json (hunk 1 above)
Comment (smiasojed) [reply LgiHdmiCec/LgiHdmiCec.json]: done

File: LgiHdmiCec/LgiHdmiCec.json (hunk 2 above)
Comment (smiasojed) [reply LgiHdmiCec/LgiHdmiCec.json]: done

**Review by michalkucharczyk — APPROVED** [review]

**Review by tomasz-karczewski-red — APPROVED** [review]

**Pull request merged**